### PR TITLE
Fix no selected categories senarios

### LINKED ISSUE #29: Items that are NOT assigned to a category can't be displayed
Category fields is mandatory due to avoid #19 

But in this way items with no category are left out.

Possible fix make category NOT mandatory but check if array is empty in the code.

### PR DESCRIPTION
**Pull request by BreakSecurity**
Second PR ;) Fix #29 hopefully without breaking anything else

### CODE REVIEW

**Comment by ryanmitchell** [comment]
Added the ability to re-blank categories.
Let me know if this works for you.

**Comment by BreakSecurity** [comment]
It still allow me to create a view with no locations selected

**Comment by ryanmitchell** [comment]
You never mentioned locations ;)
Try the latest commit

**Comment by BreakSecurity** [comment]
At the moment is possible to create multiple view with the same name should we prevent this or is too much?

File: models/Views.php [hunk 1]
```diff
@@ -27,71 +27,75 @@ class Views extends Model
...
+        'name' => 'required',
         'locations' => 'required',
         'display.order_count' => 'required|int|min:0',
         'display.refresh_interval' => 'required|int|min:10',
     ];
-    
+
     public function beforeSave()
     {
+        if (!Request::input('View.name'))
```
Comment (ryanmitchell) [inline models/Views.php:40]: 41-42 arent needed for strings

**Review by ryanmitchell — CHANGES_REQUESTED** [review]
Going forward, try to keep PRs to one bit of functionality or requirement to make for easier merging. Thanks.

**Comment by BreakSecurity** [comment]
So for me is ok to merge like this. Thanks for the tips

**Review by ryanmitchell — APPROVED** [review]

**Pull request merged**